LoopKitUI/Views/Settings Editors/GuardrailExtensions.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  GuardrailExtensions.swift
```

Comment:
[nit] I've noticed the name convention for extensions tends to be [thing you're extending]+[description]. Perhaps `Guardrail+Settings`?

Reply:
@mpangburn had a similar comment for `TherapySettingExtensions`.  I'll change both to `+Settings` suffix.